<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Producer</class>
 <widget class="QWidget" name="Producer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="producerBox">
     <property name="title">
      <string>Producer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="productCountLayout">
        <item>
         <widget class="QLabel" name="productCountLab">
          <property name="text">
           <string>Product Count: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="productSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="productDelayLab">
          <property name="text">
           <string>Product Emission Delay: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="productDelaySpinBox">
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="producePB">
        <property name="text">
         <string>Produce</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
